tools/mmp/mmp.csproj

```diff
@@ -314,6 +313,9 @@
     <Compile Include="..\..\src\ObjCRuntime\PlatformAvailability2.cs">
       <Link>external\PlatformAvailability2.cs</Link>
     </Compile>
+    <EmbeddedResource Include="config_mobile">
+       <LogicalName>config_mobile</LogicalName>
+    </EmbeddedResource>
```

Comment:
This looks like XS just moved one node to somewhere else in the file, and the changes to the entire file can be ignored.